src/resources/language/_language-fr.yml

```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Équation"
 crossref-lof-title: "Liste des Figures"
 crossref-lot-title: "Liste des Tables"
 crossref-lol-title: "Liste des Listings"
+
+listing-page-order-by: "Trier par"
+listing-page-order-by-default: "Présélectionné"
```

Comment:
Default basically represents the original order. In the event of a listing populated from files, it will be in title order. In the event of a listing from Metadata, it will just be the order of the items in the metadata itself - its a subtle difference that may be lost on users. I might be able to be smarter and only include 'default' when the original sort order isn't in the list below...

Reply:
I think that would be great, thank you!